src/index.js

```diff
@@ -1,13 +1,126 @@
-export default function BaseballGame() {
-  this.play = function (computerInputNumbers, userInputNumbers) {
-    return "결과 값 String";
-  };
-}
-
-// export default class BaseballGame {
-//   play(computerInputNumbers, userInputNumbers) {
-//     return "결과 값 String";
-//   }
-// }
-
-new BaseballGame();
+export default class BaseballGame { 
+
+    constructor() {
+      this.computerNum = this.randomNumGenerator();
+      this.strike = 0;
+      this.ball = 0;
+      this.result = '';
+      console.log(this.computerNum);
+    }
+  
+    randomNumGenerator() {
+      let first = Math.floor(Math.random() * 9 + 1);
+      let second = Math.floor(Math.random() * 9 + 1);
+      while (first == second) {
+        second = Math.floor(Math.random() * 9 + 1);
+      }
+      let third = Math.floor(Math.random() * 9 + 1);
+      while (third == first || third == second) {
+        third = Math.floor(Math.random() * 9 + 1);
+      }
+      return (
+        third.toString() + second.toString() + first.toString()
+      );
+    }
+
+    checkUserInput(userInputNumbers) {
+        let userNumArray = userInputNumbers.split('');
+        if (userNumArray.length != 3) {
+            return false;
+        }
+        else if (isNaN(userInputNumbers) == true) {
```

Comment:
```suggestion
        else if (Math.isNaN(userInputNumbers) == true) {
```
> isNaN() 함수는 어떤 값이 NaN인지 판별합니다. isNaN 함수는 몇몇 일반적이지 않은 규칙을 가지고 있으므로, ECMAScript 2015에서 추가한 Number.isNaN()으로 바꾸는 편이 좋을 수도 있습니다.

- https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/isNaN